plugin/CactbotEventSource/FFXIVProcessIntl.cs

```diff
@@ -698,5 +700,26 @@ public struct SamuraiJobMemory {
         }
       }
     }
+
+    [StructLayout(LayoutKind.Explicit)]
+    public struct SageJobMemory {
+      [FieldOffset(0x00)]
+      public ushort addersgallMilliseconds; // the addersgall gauge elapsed in milliseconds, from 0 to 19999.
+
+      [FieldOffset(0x02)]
+      public byte addersgall;
+
+      [FieldOffset(0x03)]
+      public byte addersting;
+
+      [FieldOffset(0x04)]
+      private byte eukrasia_bits;
+
+      public bool eukrasia {
+        get {
+          return (eukrasia_bits & 0x1) != 0;
+        }
+      }
```

Comment:
```suggestion
      [FieldOffset(0x04)]
      private byte eukrasia;
```

If it's always 1 or 0, then I would suggest doing the above.  That way there's a single `eukrasia` field that we send.  No need to do the `0x1` mask part.